addons/event/static/src/template_reference_field/field_event_mail_template_many2one.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<templates xml:space="preserve">
+
+    <t t-name="event.mail_template_many2one_field" t-inherit="web.Many2OneField" t-inherit-mode="primary">
```

Comment:
I think the new convention is to use camelCase for template names, not 100% sure though.

Reply:
Sounds right to me :+1: 
Also applied to the field name then